core-kotlin-modules/core-kotlin-collections-map/src/test/kotlin/com/baeldung/modifyMapEntryInPlace/ModifyMapEntryInPlaceUnitTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.modifyMapEntryInPlace
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class ModifyMapEntryInPlaceUnitTest {
+
+    @Test
+    fun `modifies mutable map entry using put method`() {
+        val map = mutableMapOf("key1" to "value1", "key2" to "value2")
+        map.put("key1", "new value")
+        assertEquals("new value", map["key1"])
+    }
+
+    @Test
+    fun `modifies mutable map entry using assign operator`() {
```

Comment:
might need to update the name of this test with the name of the section.

Reply:
To match the verbiage in the draft, let's make this `set operator` instead - using pr comments to be able to update this.

```suggestion
    fun `modifies mutable map entry using set operator`() {
```